pages/linux/pi.md

```diff
@@ -0,0 +1,24 @@
+# pi
+
+> Prints 100 decimal digits of Archimedes' constant pi or a number of digits specified by an integer parameter on the command line.
```

Comment:
The command could be used for Math operations in Bash, so it's something more than display numbers:
```suggestion
> Compute decimal Archimedes' constant Pi on the command line.
```